packages/btcindexer/db/migrations/0001_initial_schema.sql

```diff
@@ -3,13 +3,13 @@ CREATE TABLE IF NOT EXISTS btc_blocks (
   hash TEXT NOT NULL,
   height INTEGER NOT NULL,
   network TEXT NOT NULL,
-  status TEXT NOT NULL DEFAULT 'new' CHECK (status IN ('new', 'scanned')), -- TODO: change it to boolean
+  processed INTEGER NOT NULL DEFAULT 0,  -- BOOL (SQLite STRICT mode uses INTEGER for booleans)
```

Comment:
lets rename it to is_scanned, the comment is unnecessary, lets just descibe what true false mean